Makefile

```diff
@@ -215,6 +220,17 @@ terraform-init:
 	az account set -s $(AZ_SUBSCRIPTION) && az account show \
 	&& terraform -chdir=terraform/$(PLATFORM) init -reconfigure -backend-config=./workspace-variables/$(DEPLOY_ENV)_backend.tfvars $(backend_key)
 
+get-cluster-credentials: set-azure-account ## make <config> get-cluster-credentials [ENVIRONMENT=<clusterX>]
+	az aks get-credentials --overwrite-existing -g ${RESOURCE_NAME_PREFIX}-tsc-${CLUSTER_SHORT}-rg -n ${RESOURCE_NAME_PREFIX}-tsc-${CLUSTER}-aks
+
+aks-console: read-cluster-config get-cluster-credentials
+	$(if $(APP_NAME), $(eval export APP_ID=$(APP_NAME)) , $(eval export APP_ID=$(CONFIG_LONG)))
+	kubectl -n ${NAMESPACE} exec -ti --tty deployment/register-${APP_ID} -- /bin/sh
+
+aks-logs: read-cluster-config get-cluster-credentials
+	$(if $(APP_NAME), $(eval export APP_ID=$(APP_NAME)) , $(eval export APP_ID=$(CONFIG_LONG)))
+	kubectl -n ${NAMESPACE} logs deployment/register-${APP_ID} --timestamps=true
```

Comment:
This shows the logs from the first pod in the deployment. To get all the pods we could use something like: `kubectl -n ${NAMESPACE} logs -l app=register-${APP_ID} --timestamps=true`

Reply:
changed to use -l